content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -215,7 +215,40 @@ Nevertheless, you must update the list of admission controllers.
 
 ### Azure
 
-You must use `ACS-Engine` to deploy your cluster.
+You can deploy a Kubernetes cluster to Azure via `ACS-Engine` or `AKS` which fully supports Istio as long as Kubernetes 1.9.6 or later is specified.
+
+AKS is a managed Kubernetes service where the master nodes are paid for and managed by Microsoft, no matter the size or number of agent nodes created.  
+
+ACS-Engine provides extensive customization of the Kubernetes cluster being provisioned.  The downloadable `acs-engine` binary processes a simple api-model JSON file into a large Azure resource manager template which automatically deploys a complete Kubernetes cluster.  Master and agent nodes are mangaged entirely and paid for by the customer.
+
+Instructions for AKS:
+
+You can create an AKS cluster via [the `az` cli](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) or [the Azure portal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal).  It is only important Kubernetes 1.9.6 or later is specified.  Not all Kubernetes versions are available in all Azure regions.
```

Comment:
Referring to the following sentence. 
 *It is only important Kubernetes 1.9.6 or later is specified.*
Consider changing it to:
*Ensure that you use Kubernetes 1.9.6 or later.*

Referring to the following sentence:
*Not all Kubernetes versions are available in all Azure regions.*

Should we be more positive and show the reader the latest supported version in each of the data centers? This can be achieved with the following command:

```
$ az provider list --query "[?namespace=='Microsoft.ContainerService'].resourceTypes[] | [?resourceType=='managedClusters'].locations[]" -o tsv | xargs -n1 -I % az aks get-versions --location "%" --query "orchestrators[].orchestratorVersion | [-1:] | { location:'%', versions: @[0] }"
```

Or for a specific data center if that is more appropriate:

```
$ az aks get-versions --location "Australia East" --query "orchestrators[].orchestratorVersion | [-1:] | @[0]"
```